### DISCUSSION TITLE
Which city hosted the Summer Olympics in 2012?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#390] [Galaxy Trivia] [answered]
**Hint:** 💂🏼‍♂️

**Comment by peterbaccaro** [comment, marked as answer]
**London** (England, UK) was the city that hosted the 2012 Summer Olympics.

**Reply by ParanoidUser** [reply]
Were you in the city during the Olympics? How did it feel? 😊

**Reply by peterbaccaro** [reply]
Yes, it was good but very difficult to get tickets for the events.

**Comment by thuongtruong1009** [comment]
The city that hosted the Summer Olympics in 2012 was London, the capital city of England and the United Kingdom. The 2012 Summer Olympics, officially known as the Games of the XXX Olympiad, were held from July 27th to August 12th, 2012, and featured over 10,000 athletes from more than 200 countries competing in 26 different sports. It was the third time that London had hosted the Summer Olympics, having previously done so in 1908 and 1948.
